src/components/InteractiveArticle.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import BaseComponent from 'lib/falcor/FalcorController';
+
+export default class InteractiveArticle extends FalcorController {
+  static getFalcorPaths(params) {
+    return [
+      // For now we only use the html part, the js and css parts are for further improvements
+      ['interactiveBySlug', params.slug, ['html', 'js', 'css']],
+    ];
+  }
+
+  render() {
+    const slug = this.props.params.articleSlug;
+    const reactHtml = {
+      __html: this.state.data.interactiveBySlug[slug].html,
+    };
+    return <div dangerouslySetInnerHTML={reactHtml} />;
```

Comment:
What I'm talking about is if we compiled the `index.html` file from https://github.com/mklasinc/im-article-boilerplate so all the files weren't distributed but all in one html file, and then injected it, it would have the `<html>` and `<head>` tags and what not. That wouldn't work out. 

Yep I know React Helmet just changes the `<head>` and `<html>` tags and I believe maybe the classnames of `body` as well or something like that.

What I was suggesting was just implementing the boilerplate in this component, as we won't actually be serving a whole static html file, only the `<body>` tag as that's what this component here will actually replace, a `<div>` that is the direct child of the `<body>` tag. You're talking about the navbar, as of right now, with current implementation, no it won't display the nav bar, which was very much on purpose. Do you want it to? I thought we were doing a different layout thing like NY times, and then had a specific header for the interactive articles that had a different more minimalist design or something like that. And that's what I talked about in terms of the benefits of if we wanted to change the interactive header for any reason in the future as things and preferences change, it would be neat because it would automatically apply to past articles.